clients/admin-ui/src/constants.ts

```diff
@@ -77,11 +77,19 @@ export const USER_PRIVILEGES: UserPrivileges[] = [
     scope: "user:create",
   },
   {
-    privilege: "Create roles",
+    privilege: "Reset their own password",
+    scope: "user:reset-password",
```

Comment:
This will allow users to have the options to reset their own passwords.

Reply:
Did they not have this before? Is there any way to have this default to `true`?

Reply:
We can do that too in a follow-up 👍